portfolio-project-matching-app/pages/index.js

```diff
@@ -1,38 +1,29 @@
 import Head from 'next/head'
 import Image from 'next/image'
 import DBList from '../components/DBList'
-import { db } from '../Firebase/clientApp.ts'
-// import firebase from 'firebase/app'
-import { collection, getDocs } from 'firebase/firestore'
-// import { useCollection } from 'react-firebase-hooks/firestore'
+import DBForm from '../components/DBForm'
+import { getAllDocs } from '../Firebase/clientApp.ts'
 
-export default function Home({ docList }) {
+export default function Home({ projectList }) {
```

Comment:
Most of the changes in here are negligible. At the moment, we are just using `index.js` as a sandbox for displaying and tinkering with our components. As we build out our pages and routing, we won't be messing with this stuff as much.